src/components/WorkspaceDetails/WorkspaceDetails.tsx

```diff
@@ -21,6 +23,21 @@ const WorkspaceDetailsBreadcrumbs = (workspaceName: string) => [
   { name: 'Workspace Details', path: workspaceName ? `/workspaces/${workspaceName}` : '' },
 ];
 
+const WorkspaceActionMenu = (setDeleteModalOpen: (isOpen: boolean) => void) => ({
+  actions: [
+    {
+      id: '2',
```

Comment:
NIt - can we either remove ID or change it to something more specific?

Reply:
For now, let's just remove it